### ISSUE TITLE
Do not mimimize a docker image during conversion

### ISSUE THREAD

**Issue by Pigrenok** [ktock/container2wasm#222]
Hello!

It is more of a feature request rather than an issue.

I am converting a custom image (built on top of ubuntu:22.04 docker image) using command c2w as follows:
```
c2w custom-image:latest out.wasm
```
This custom-image is built in such a way that man command works and even sudo command is available for the user (also created in Dockerfile for the custom-image).

But after conversion I get the following:
- When I run `man` command, I get 
```
This system has been minimized by removing packages and content that are
not required on a system that users do not log into.
    
To restore this content, including manpages, you can run the 'unminimize'
command. You will still need to ensure the 'man-db' package is installed.
```
- When I try to use `sudo -s`, I get
```
sudo: The "no new privileges" flag is set, which prevents sudo from running as root.
sudo: If sudo is running in a container, you may need to adjust the container configuration to disable the flag.
```

Both commands work fine if I just run this image in a docker container (via `docker run ...`).

I can probably live without `sudo`, but `man` is a necessity for my project.

I assume the image is being modified during conversion. Is it possible avoid this conversion and tell c2w command to just use the image as is without any modifications. Or there is a reason for this that cannot be avoided?

Thank you very much in advance for your help.

**Comment by ktock** [comment]
Thanks for reporting this. Could you provide a minimal Dockerfile that reproduces this issue?

**Comment by Pigrenok** [comment]
Here is the Dockerfile that I use to produce my image
```
FROM ubuntu:22.04

RUN apt-get clean && apt-get update && apt-get -y upgrade

RUN yes | unminimize

RUN apt-get -y install apt-utils curl \
	less nano tree perl file sudo\
	man man-db manpages manpages-posix
RUN apt autoremove && apt clean

RUN useradd -m learner && echo "learner:learner" | chpasswd
RUN echo 'learner ALL=(ALL:ALL) NOPASSWD: ALL' >> /etc/sudoers

WORKDIR /home/learner/

USER learner

CMD [ "/bin/bash" ]
```

**Comment by ktock** [comment]
Fixed in #225
